src/test/java/nextstep/ladder/LadderTest.java

```diff
@@ -0,0 +1,34 @@
+package nextstep.ladder;
```

Comment:
```suggestion
package nextstep.ladder.domain;
```
테스트 패키지의 경로를 main 과 동일하게 해주면 어떨까요? 😄
main 에서 접근제어자를 default 로 선언해주게 되면 동일한 테스트 패키지에서 접근이 가능하답니다!